docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -0,0 +1,883 @@
+   |                      | Can be run, at most, once per experiment.                  |
+   +----------------------+------------------------------------------------------------+
+   | get_extrn_ics        | Cycle-specific task to obtain external data for the        |
+   |                      | initial conditions                                         |
+   +----------------------+------------------------------------------------------------+
+   | get_extrn_lbcs       | Cycle-specific task to obtain external data for the        |
+   |                      | lateral boundary conditions (LBC's)                        |
+   +----------------------+------------------------------------------------------------+
+   | make_ics             | Generate initial conditions from the external data         |
+   +----------------------+------------------------------------------------------------+
+   | make_lbcs            | Generate LBC's from the external data                      |
+   +----------------------+------------------------------------------------------------+
+   | run_fcst             | Run the forecast model (UFS weather model)                 |
+   +----------------------+------------------------------------------------------------+
+   | run_post             | Run the post-processing tool (UPP)                         |
+   +----------------------+------------------------------------------------------------+
+
+
+
+.. _RocotoRun:
+
+Run the Workflow Using Rocoto
+=============================
+The information in this section assumes that Rocoto is available on the desired platform. (Note that Rocoto cannot be used when running the workflow within a container.) If Rocoto is not available, it is still possible to run the workflow using stand-alone scripts according to the process outlined in :numref:`Section %s <RunUsingStandaloneScripts>`. There are two main ways to run the workflow with Rocoto: (1) with the ``launch_FV3LAM_wflow.sh`` script, and (2) by manually calling the ``rocotorun`` command. Users can also automate the workflow using a crontab. 
+
+Optionally, an environment variable can be set to navigate to the ``$EXPTDIR`` more easily. If the login shell is bash, it can be set as follows:
+
+.. code-block:: console
+
+   export EXPTDIR=/<path-to-experiment>/<directory_name>
+
+If the login shell is csh/tcsh, it can be set using:
+
+.. code-block:: console
+
+   setenv EXPTDIR /path-to-experiment/directory
+
+
+Launch the Rocoto Workflow Using a Script
+-----------------------------------------------
+
+To run Rocoto using the ``launch_FV3LAM_wflow.sh`` script provided, simply call it without any arguments: 
+
+.. code-block:: console
+
+   cd $EXPTDIR
+   ./launch_FV3LAM_wflow.sh
+
+This script creates a log file named ``log.launch_FV3LAM_wflow`` in ``$EXPTDIR`` or appends information to it if the file already exists. Check the end of the log file periodically to see how the experiment is progressing:
+
```

Comment:
There's also the FV3LAM_wflow.log file in the /log directory, which shows Rocoto task information.